stablehlo/reference/Ops.h

```diff
@@ -83,6 +83,8 @@ Tensor evalSelectOp(const Tensor &pred, const Tensor &onTrue,
 Tensor evalSineOp(const Tensor &operand, TensorType resultType);
 Tensor evalSliceOp(const Tensor &operand, Index startIndices, Sizes strides,
                    TensorType resultType);
+SmallVector<Tensor> evalSortOp(ArrayRef<Tensor> inputs, Axis dimension,
+                               bool isStable, Region &comparator, Scope &scope);
```

Comment:
Should the signature also include `resultTypes`, similarly to all other eval functions in this file? Same question for `evalWhileOp`.

Reply:
I've been thinking about this some time back, but decided it's not necessary to include it since there is no immediate need for this parameter. Though it does not align perfectly with the rest, but for example, IfOp, CastOp, and ConstantOp don't have `resultType` either, and we haven't run into issues so far.